cluster/common.sh

```diff
@@ -288,12 +288,7 @@ function load-or-gen-kube-basicauth() {
 #   KUBE_VERSION
 function set_binary_version() {
   if [[ "${1}" =~ "/" ]]; then
-    IFS='/' read -r -a path <<< "${1}"
-    if [[ "${path[0]}" == "release" ]]; then
-      KUBE_VERSION=$(gsutil cat "gs://kubernetes-release/${1}.txt")
-    else
-      KUBE_VERSION=$(gsutil cat "gs://k8s-release-dev/${1}.txt")
-    fi
```

Comment:
@jprzychodzen do we know if these files are used elsewhere and there is risk of breaking some internal pipelines?

Reply:
AFAIK there is no usage of `/cluster` internally - it's more like a test code to run CCM E2E tests.